src-sample/main/default/aura/SampleLookupContainer/SampleLookupContainer.cmp

```diff
@@ -1,7 +1,11 @@
 <aura:component access="global" controller="SampleLookupController">
     <aura:attribute access="private" type="List" name="selection" default="[]"/>
+    <aura:attribute access="private" type="object[]" name="errors" default="[]"/>
 
-    <div class="slds-form slds-form_stacked slds-m-around_xx-large">
-        <c:Lookup aura:id="lookup" selection="{!v.selection}" onSearch="{!c.lookupSearch}" label="Search" placeholder="Search Salesforce" isMultiEntry="true"/>
+    <div class="slds-form slds-form_stacked slds-m-around_xx-large"></div>
```

Comment:
Syntax error: that closing `div` breaks the component.

Reply:
Whoops, that probably happened when pasting it in from the other file I was working in to test the sample code. VS Code autocomplete goes wrong...